nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -23,11 +31,12 @@
           skip-tag: true
           wipe-workspace: true
           basedir: "nfs-ganesha-debian"
-- job:
-    name: nfs-ganesha
+
+- job-template:
```

Comment:
yes, that's right. here's some good docs on how ``project`` and ``job-template`` works together: https://docs.openstack.org/infra/jenkins-job-builder/definition.html#job-template